### PR TITLE
Styles

### PR DESCRIPTION
**Pull request by Anteycu**

### CODE REVIEW

File: src/components/customNavLink.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { NavLink } from "react-router-dom";
+
+interface Props {
+  children: string;
```
Comment (lluchkaa) [inline src/components/customNavLink.tsx:4]:
```suggestion
  children: React.ReactNode;
```

File: src/components/featureItem.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+interface Props {
+  children: string;
```
Comment (lluchkaa) [inline src/components/featureItem.tsx:2]:
```suggestion
  children: React.ReactNode;
```

File: src/components/featureItem.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  title: string;
+  src: string;
+  order?: string;
+}
+
+export const FeatureItem = ({ children, title, src, order }: Props) => {
+  return (
+    <div
+      className={`flex ${order} justify-around items-center mb-20 text-xl font-medium text-yellow-900`}
```
Comment (lluchkaa) [inline src/components/featureItem.tsx:11]: You can replace `order` with `className`. Also, use the example from the comment on the `button.ts` file:

[Watch video](https://youtu.be/re2JFITR7TI?si=o0SkTTuWJRRRVgeB)

```suggestion
      className={cn("flex justify-around items-center mb-20 text-xl font-medium text-yellow-900", className)}
```

File: src/components/featureItem.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    >
+      <img className="w-1/3 rounded-4xl" src={src} alt="Barber works" />
+      <div className="w-1/3">
```
Comment (lluchkaa) [inline src/components/featureItem.tsx:14]: There are two blocks, but each takes 33.3% of width. Consider using `w-1/2` for one of them

File: src/components/customNavLink.tsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  to: string;
+}
+
+export const CustomNavLink = ({ children, to }: Props) => {
```
Comment (lluchkaa) [inline src/components/customNavLink.tsx:8]: Hmm, what about renaming it to `NavigationLink`?

File: src/components/footer.tsx
Comment (lluchkaa) [inline src/components/footer.tsx:1]: As there is a lot of repeating things here, consider moving data to a constant list, then map it in the component.

`components/footer/constants.ts`
```tsx
const LINKS = [
  {
    title: "Features",
    links: [
      {
        text: "Calendar",
        href: "/calendar"
      },
      {
        text: "Bookings",
        href: "/bookings"
      }
    ]
  },
  {
    title: "About",
    links: [
      {
        text: "Team",
        href: "/team"
      }
    ]
  }
]
```

`components/footer/index.tsx`
```tsx
export const Footer = () => {
  return (
    <footer className="pt-8 pb-8">
      <Container>
        <div className="flex gap-x-80 pl-20 text-lg font-medium text-yellow-900">
          {LINKS.map(({ title, links }) => (
            <div>
              <h3>{title}</h3>
              <ul className="list-none p-0 text-base">
                {links.map(({ text, href }) => (
                  <li>
                    <a
                      className="text-yellow-950 no-underline hover:text-yellow-600"
                      href={href}
                    >
                      {text}
                    </a>
                  </li>
                ))}
              </ul>
            </div>
          ))}
        </div>
      </Container>
    </footer>
  )
```

File: src/components/logo.tsx [hunk 6]
```diff
@@ -0,0 +1,9 @@
+export const Logo = () => {
+  return (
+    <a>
```
Comment (lluchkaa) [inline src/components/logo.tsx:3]: Use this as a link to main page

File: src/pages/BarberDetails.tsx [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+import { Link, useParams } from "react-router-dom";
+import { Barber } from "../domains/barber/types";
+import { Button } from "../components/button";
+import { Container } from "../components/container";
+
+export const BarberDetails = () => {
+  const { id } = useParams();
+  const barber: Barber | undefined = getBarberById(id);
+  if (barber) {
+    return (
```
Comment (lluchkaa) [inline src/pages/BarberDetails.tsx:11]: Use early return here:

```tsx
if (!barber) {
  return <p>No barber</p>
}

return (
  ...
)
```

File: src/pages/About.tsx
Comment (lluchkaa) [inline src/pages/About.tsx:1]: You can follow the same instruction as in `footer.tsx` page. You can move data to constant, and then map it in the component.

**Review by lluchkaa — APPROVED** [review]

**Review by anastasiia-tarasenko — APPROVED** [review]
I ENJOY YOUR STYLES 🎉

File: src/components/form.tsx [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+  const handleSubmit = (evt: React.SyntheticEvent) => {
+    evt.preventDefault();
+    const clientInfo = {
+      name: firstName,
+      surname: secondName,
+      email: email,
+      order: service,
+    };
+    localStorage.setItem("clientInfo", JSON.stringify(clientInfo));
+    reset();
```
Comment (lluchkaa) [inline src/components/form.tsx:25]: After saving the info, you can use `useNavigate` hook to redirect user to another page.
[Learn more here](https://reactrouter.com/en/main/hooks/use-navigate)

```tsx
const navigate = useNavigate()

...
const handleSubmit = () => {
  ...
  redirect('boooking')
}
```

**Pull request merged**